src/infiniteLoops/runtime.ts

```diff
@@ -268,6 +268,7 @@ function prepareBuiltins(oldBuiltins: Map<string, any>) {
     }
   }
   newBuiltins.set('undefined', undefined)
+  newBuiltins.set('Object', Object)
```

Comment:
Not sure if this is the right move; what are the implications of this? Perhaps someone could verify/advise.

Reply:
I see that you are updating the name property in the function object. But here, are you trying to add the name "Object" to the names of builtins that are visible from Source programs?